app/assets/v2/css/gitcoin.css

```diff
@@ -71,6 +71,7 @@ h4,
   background-position: 0px 0px;
   background-repeat: no-repeat;
   position: relative;
+  z-index: 3;
```

Comment:
why is this needed?

Reply:
this is used over the site in some places the next box need to overlap the header, for example on onboard.

Reply:
good catch, this change was from another PR isn't required here.